secrets.properties

```diff
@@ -0,0 +1 @@
+GOOGLE_MAPS_API_KEY = AIzaSyAHkGMX1AGthHeTC7LU8sKfyEAJ2danMgo
```

Comment:
Does not everybody can see this change ? I could see it when not connected in the PR. Is it possible to not push this file ?